### PR TITLE
LRN 17472 - Update to .NET Standard

### PR DESCRIPTION
**Pull request by gonzalozawa**
- Remove legacy solution file: LearnositySDK-VS2010.sln
- Update to .Net Standard
- Update Newtonsoft dependency to version 10.0.3 - This was needed when updating to .Net Standard
- Change LearnositySDK project to a Class Library that targets .NET Standard framework 2.0
- Update LearnositySDK project to build a Nuget package

### CODE REVIEW

**Comment by gonzalozawa** [comment]
Hi @karoltarasiuk, I have resolved the conflicts on this branch with master. Can you please have a look now and review?
With this update, my idea is to upgrade the Nuget package (https://www.nuget.org/packages/LearnositySDK/) with this latest changes to target .NET Standard.
Please let me know your comments.

File: LearnositySDK/LearnositySDK.csproj [hunk 1]
```diff
@@ -1,83 +1,25 @@
...
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProductVersion>8.0.30703</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>{4A1F8BE2-02A2-424D-AE33-11F742E18EED}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>LearnositySDK</RootNamespace>
+    <TargetFramework>netstandard2.0</TargetFramework>
```
Comment (karoltarasiuk) [inline LearnositySDK/LearnositySDK.csproj]: @gonzalozawa what did you pick the version of .NET standard based on? They recommend:
> to target the lowest version of .NET Standard possible

Comment (gonzalozawa) [reply LearnositySDK/LearnositySDK.csproj]: @karoltarasiuk this is a great question. I've tried using a lower version of .NET Standard but the project doesn't compile and throw errors. Mainly on Remote.cs and Init.cs
The problem in that particular classes is that System.Web and System.Net aren't included in the previous version of .NET Standard. See https://stackoverflow.com/questions/30227219/how-to-use-webclient-with-netcore
Do you think I should try to fix those classes and use .NET Core functionality or should we just target .NET Standard 2.0 ?

Comment (karoltarasiuk) [reply LearnositySDK/LearnositySDK.csproj]: Cool, makes sense. I think that in this case we leave it as is and only come back to it if somebody complains :)

**Comment by karoltarasiuk** [comment]
@gonzalozawa I think all we need then is a squash and merge then.

**Pull request merged**